crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -319,45 +312,63 @@ mod tests {
         let calculated_len_3 = get_deposit_byte_len(&deposit);
         assert!(calculated_len_3 >= calculated_len_2);
 
-        // Ensure calculated length is still greater than or equal to actual with absurd string
+        // Ensure calculated length is as expected with absurd string
         // lengths (have tested up to 99999999, but this makes testing very slow)
         let absurd_string: String = ['a'; u16::MAX as usize].iter().collect();
         let deposit = Deposit::new(
             astria_address(&[1; 20]),
             RollupId::from_unhashed_bytes(b"test_rollup_id"),
             u128::MAX,
             absurd_string.parse().unwrap(),
-            absurd_string,
+            absurd_string.clone(),
             TransactionId::new([0; 32]),
             u64::MAX,
         );
         let calculated_len = get_deposit_byte_len(&deposit);
-        let expected_len = deposit.into_raw().encoded_len() as u128;
-        assert!(calculated_len >= expected_len);
+        let expected_len = 16 + u128::from(u16::MAX) * 2;
+        assert_eq!(calculated_len, expected_len);
+    }
+
+    #[test]
+    fn get_deposit_byte_length_snapshot() {
+        let deposit = Deposit::new(
+            astria_address(&[1; 20]),
+            RollupId::from_unhashed_bytes(b"test_rollup_id"),
+            u128::MAX,
+            test_asset(),
+            "someaddress".to_string(),
+            TransactionId::new([0; 32]),
+            u64::MAX,
+        );
+        assert_json_snapshot!(get_deposit_byte_len(&deposit));
     }
 
     /// Used to determine the base deposit byte length for `get_deposit_byte_len()`. This is based
-    /// on usage of the longest allowed values for `bridge_address`, `amount`, and
-    /// `source_action_index`. `asset` and `destination_chain_address` are empty strings, whose
-    /// length will be added to the base cost at the time of calculation.
+    /// on "reasonable" values for all fields except `asset` and `destination_chain_address`. These
+    /// are empty strings, whose length will be added to the base cost at the time of
+    /// calculation.
+    ///
+    /// This test determines 165 bytes for an average deposit with empty `asset` and
+    /// `destination_chain_address`, which is divided by 10 to get our base byte length of 16. This
+    /// is to allow for more flexibility in overall fees (we have more flexibility multiplying by a
+    /// lower number, and if we want fees to be higher we can just raise the multiplier).
     #[test]
     #[ignore]
     fn get_base_deposit_byte_length() {
```

Comment:
this dest should not be `ignore` and it should test that `DEPOSIT_BASE_BYTE_LENGTH` is the result.